### PR TITLE
Update travis to trusty

### PR DESCRIPTION
**Pull request by pieterlexis**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,49 +22,55 @@ before_script:
...
+     libboost-all-dev
      libbotan1.10-dev
      libcdb-dev
-     libcdb-dev
      libcurl4-openssl-dev
+     libgeoip-dev
      libldap2-dev
-     liblua5.1-posix1
+     liblmdb-dev
+     liblmdb0
```
Comment (Habbie) [inline .travis.yml]: It looks like you accidentally reintroduced installation of liblmdb&co

File: .travis.yml [hunk 2]
```diff
@@ -23,49 +22,55 @@ before_script:
...
      unbound-host
      xmlto
- - sudo sh -c 'sed s/precise/trusty/g /etc/apt/sources.list > /etc/apt/sources.list.d/trusty.list'
- - sudo apt-get update --quiet --quiet
- - sudo apt-get install --quiet --quiet libyaml-cpp-dev libzmq3-dev pandoc libgeoip-dev
- - sudo pip install pandocfilters==1.2.3 mkdocs==0.14 linkchecker==9.3 click==5.1
- - sudo update-alternatives --set ruby /usr/bin/ruby1.9.1
+ - virtualenv $HOME/.venv
+ - source $HOME/.venv/bin/activate
+ - pip install pyopenssl ndg-httpsclient pyasn1
```
Comment (Habbie) [inline .travis.yml]: we need both `python-openssl` (under `apt-get`) and the one in the venv?

**Comment by Habbie** [comment]
only nits. LGTM!

File: .travis.yml (hunk 1 above)
Comment (pieterlexis) [inline .travis.yml]: will remove

File: .travis.yml (hunk 2 above)
Comment (pieterlexis) [inline .travis.yml]: just to be sure, `pip` was giving me a bad time installing pandocfilters because of requests complaing about ssl.

Comment (Habbie) [inline .travis.yml]: :+1:

**Pull request merged**